src/Components/ImportModal/ImportModal.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { useState } from "react";
+import { ViewMedia, DragModal } from "./ViewMedia";
+import styles from "./ImportModal.module.css";
+import { Button } from "react-bootstrap";
+
+type Props = {
+  setShow: React.Dispatch<React.SetStateAction<boolean>>;
+  removeFile: (index: number) => void;
+  addFile: (newMedia: File[]) => void;
+};
+
+export const ImportModal: React.FC<Props> = ({
+  setShow,
+  removeFile,
+  addFile,
+}) => {
+  const createMediaElement = (files: File[]) => {
+
```

Comment:
For future PRs: if you have a placeholder fn which is still to be implement, then add a TODO comment, otherwise it looks like redundant code